open-api-spec.yaml

```diff
@@ -784,31 +948,31 @@ components:
         id:
           type: string
           format: uri
-          description: The URL identifying the incoming payment.
+          description: The URL identifying the outgoing payment.
           example: 'https://openpayments.guide/alice/outgoing-payments/8c68d3cc-0a0f-4216-98b4-4fa44a6c88cf'
           readOnly: true
         accountId:
           type: string
           format: uri
-          description: The URL of the account this payment is being made into.
+          description: The URL of the account from which this payment is sent.
           example: 'https://openpayments.guide/alice/'
           readOnly: true
+        quoteId:
+          type: string
+          format: uri
+          description: The URL of the quote defining this payment's amounts.
+          example: 'https://openpayments.guide/alice/quotes/ab03296b-0c8b-4776-b94e-7ee27d868d4d'
+          readOnly: true
         state:
           type: string
           description: The state of the payment as described above.
           enum:
             - pending
+            - processing
```

Comment:
Could `processing` be removed? Without authorization, the distinction between `pending` and `processing` seems implementation specific.
Maybe `processing` could also be removed from incoming payment state, since you can check the `receivedAmount` of a `pending` incoming payment.

Reply:
Correct, although an outgoing payment can alse be `failed`.

>Partial sent/received funds should not change the state?

Currently outgoing payment state changes to `processing` *before* starting to send, not after the first successfully sent packet (like incoming payments).
If we add a `sentAmount` to outgoing payments, we could similarly look at that for a `pending` payment, like I'm suggesting we do with an `pending` incoming payment's `receivedAmount`.

Reply:
As per our discussion yesterday, are you including the changes to remove `state`?

Reply:
https://github.com/interledger/open-payments/pull/97/commits/9a91081daf5250e49075a4e226e2afdd2b7611f8